uiowa/uiowa
# 2plus2 Admissions - Available Majors as a block

# How to test
`composer install`
`blt frontend`
`ddev blt sync --site admissions.uiowa.edu --define drush.aliases.remote=admissions.dev`
Go to https://admissions.uiowa.ddev.site/academics/2plus2
- Verify that participating colleges and major blocks are displaying
- Click through colleges and majors and verify that breadcrumb paths are all the same and displaying on each 2plus2 page. 
- Verify that urls are set to `academics/2plus2/title`
